File: 1st/Jinseon/길이에따른연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = num_list.length;
        if(answer > 10){
            answer = 0;
            for(int i = 0; i < num_list.length; i++){
                answer += num_list[i];
            }
        } else {
            answer = 1;
            for(int i = 0; i < num_list.length; i++){
                answer *= num_list[i];
            }
        }
        return answer;
    }
}

File: 3st/Jaeyoung/길이에 따른 연산.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    int s = 0;
    int m = 1;
    for( auto x : num_list ) {
        if( num_list.size() >= 11 ) {
            s += x;
            answer = s;
        } // if
        else {
            m *= x;
            answer = m;
        } // else
    } // for
    
    return answer;
}